sklearn/ensemble/_forest.py

```diff
@@ -512,7 +513,7 @@ def _compute_oob_predictions(self, X, y):
                 (n_samples, 1, n_outputs)
             The OOB predictions.
       """
-        X = self._validate_data(X, dtype=DTYPE, accept_sparse="csr", reset=False)
+        X = check_array(X, dtype=DTYPE, accept_sparse="csr", copy=False)
```

Comment:
I feel like we either want to validate the data or not. If this is a private method and we know validation is done somewhere else, why do we need to call `check_array`?